doc/releases/quincy.rst

```diff
@@ -10,6 +10,19 @@ v17.2.9 Quincy
 
 This is the ninth backport (hotfix) release in the Quincy series. We recommend that all users update to this release.
 
+Users should expect to see the el8 rpm subdirectory empty and the "dnf" commands are expected
+to fail with 17.2.9.
+They can choose to use 17.2.9 RPM packages for centos 8/el8 provided by CERN as a community
+member or continue to stay at 17.2.7 following instructions
```

Comment:
"continue to stay" is a pleonasm, suggest s/continue to //

Reply:
Addressed in [f575191](https://github.com/ebocchi/ceph/commit/f575191c5d9687bab2e8ae7bcd21642c812b5931)